bin/consul-cli.rb

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env ruby
+# rubocop:disable all
+
+require 'diplomatic_bag'
+require 'optparse'
+
+def syntax
+  printf "USAGE: #{__FILE__} [[action]] [[options]]\n\n" \
+          "#{__FILE__} services list [options]\n" \
+          "#{__FILE__} service status <service-name> [options]\n" \
+          "#{__FILE__} info [options]\n" \
+          "#{__FILE__} nodes list-duplicate-id [options]\n\n" \
+          "Options:\n" \
+          "-c, --consul-server\n" \
+          "-d, --datacenter\n" \
```

Comment:
same, use -dc

Reply:
-dc is not compatible with optparse.
--datacenter match the consul option syntax (-d for short)